windows/win_file.py

```diff
@@ -56,17 +56,26 @@
 
 EXAMPLES = '''
 # create a file
-- win_file: path=C:\\temp\\foo.conf 
+- win_file:
+    path: C:\temp\foo.conf
 
 # touch a file (creates if not present, updates modification time if present)
-- win_file: path=C:\\temp\\foo.conf state=touch 
+- win_file:
+    path: C:\temp\foo.conf
```

Comment:
I generally hate doc fragments, but building one that just links to a page describing all the different ways to deal with Windows paths (only for Windows modules that have path args) might not be a bad thing. I'd rather not put all that content inline though. I'm doing a bunch of Windows doc work, so I can add that page to my list of things to build.